core/src/api/Account.hpp

```diff
@@ -9,7 +9,7 @@
 #include <memory>
 #include <string>
 #ifndef LIBCORE_EXPORT
-    #if defined(_MSC_VER)
+    #if defined(_MSC_VER) && _MSC_VER <= 1900
```

Comment:
Please update your djinni before regenerating the bindings, it seems that you have an old version !